upgrade-db/Migrations/M_34.hs

```diff
@@ -0,0 +1,46 @@
+--
+-- Copyright (c) 2017 Assured Information Security, Inc. <lejosnej@ainfosec.com>
+-- 
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 2 of the License, or
+-- (at your option) any later version.
+-- 
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+-- 
+-- You should have received a copy of the GNU General Public License
+-- along with this program; if not, write to the Free Software
+-- Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+--
+
+{-# LANGUAGE PatternGuards #-}
+
+-- description: Adjust VM settings for XL
+-- date: 02/03/2017
+
+module Migrations.M_34 (migration) where
+
+import UpgradeEngine
+import Data.List
+
+migration = Migration {
+              sourceVersion = 34
+            , targetVersion = 35
+            , actions = act
+            }
+
+act :: IO ()
+act = do
+  updateGuests
+
+updateGuests = xformVmJSON xform where
+  xform tree = case jsGet "/type" tree of
+    Just s | jsUnboxString s == "svm" -> modify tree
+    _ -> tree
+    where
+      modify = jsSet "/v4v-firewall-rules/4" (jsBoxString "my-stubdom -> 0:5100") .
```

Comment:
I'll try... See https://openxt.atlassian.net/browse/OXT-1088
In the meantime, I think this PR should still go in, to enable upgrade testing.
Thanks